server/handler.go

```diff
@@ -21,21 +29,35 @@ type Handler struct {
 	mu sync.Mutex
 	e  *sqle.Engine
 	sm *SessionManager
+	c  map[uint32]*mysql.Conn
```

Comment:
there is some logic to do that in the `SessionManager`, which already kind of handled that. If this is moved here, maybe we should revisit `SessionManager` and remove the logic associated to that

Reply:
SessionManager deals with sessions but does not track connections. If we want to support `KILL CONNECTION` I can not find a better place to store connections.

Reply:
I've reread this and the SessionManager and I believe I understand you know :sweat_smile:. Are you suggesting to store the connection in the session or the SessionManager?

Reply:
Actually, after a second look, I think it's OK like this. SessionManager should only track the sessions.